Formula/ipinfo-cli.rb

```diff
@@ -0,0 +1,21 @@
+class IpinfoCli < Formula
+  desc "Official CLI for the IPinfo IP Address API"
+  homepage "https://ipinfo.io/"
+  url "https://github.com/ipinfo/cli/archive/ipinfo-1.1.2.tar.gz"
+  sha256 "4172c686b15d1a92f39b3490eea4f56325d7b5f9e6696c1d12c4aedc9706a2d6"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+
+  conflicts_with "ipinfo",
+    because: "homebrew/core/ipinfo conflicts with the binary name provided by this formula"
+
+  def install
+    system "./ipinfo/build.sh"
+    bin.install "build/ipinfo"
+  end
+
+  test do
+    assert_equal "1.1.2\n", `#{bin}/ipinfo version`
```

Comment:
I understand that this doesn't look like a useful test. But there's no way this test would pass if there was a build issue, as it's a Golang binary.

Reply:
Can we add an additional test case anyway? `test` blocks aren't just for testing that the build succeeded, but to make sure `brew`'s relocation code interacts properly with the formula. It can be a very simple test.

Reply:
> But there's no way this test would pass if there was a build issue, as it's a Golang binary.

You'd be surprised how often things are broken that could not have been caught by a compiler.

Reply:
Alright, I'll add a test that runs some of its internal IP-printing logic, which should hopefully be sufficient.

Reply:
https://github.com/Homebrew/homebrew-core/pull/77280/commits/2776e7293c9a070e3b5413a6cb453201d9282eff